README.md

```diff
@@ -0,0 +1,30 @@
+
+![](./images/MB.png)
+
+# System Overview.
+
+The Motherboard has seven identical slots for ![](https://latex.codecogs.com/gif.latex?35mm%20%5Ctimes%2040mm) modules which
+are connected to a single CPU module slot which has up to seventy general purpose
+inputs and outputs (GPIO), as well as ![](https://latex.codecogs.com/gif.latex?SPI), ![](https://latex.codecogs.com/gif.latex?I%5E%7B2%7DC), and ![](https://latex.codecogs.com/gif.latex?USB) communication buses.
```

Comment:
Although, I'm a fan of latex, if you just want to use it to create *italics* or **strong emphasis**, putting it between  asterisks, or underscores (single our double respectively) also works.